internal/actor/certificate_provider_provided.go

```diff
@@ -31,6 +31,8 @@ type CertificateProviderProvidedDetails struct {
 	Tasks CertificateProviderTasks
 	// ContactLanguagePreference is the language the certificate provider prefers to receive notifications in
 	ContactLanguagePreference localize.Lang
+	// Email is the email address returned from OneLogin when the certificate provider logged in
+	Email string
```

Comment:
was this meant to be MLPAB-1546, and so this needs sending to the lpa-store?

Reply:
My bad - I'd looked at the wrong column when grabbing the ticket number. Updated now.